pandamium_datapack/data/pandamium/functions/triggers/particles.mcfunction

```diff
@@ -1,6 +1,48 @@
+execute if score @s particles matches 1.. run tellraw @s [{"text":"19: ","color":"green","bold":false},{"text":"[Crying Obsidian]  ","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -19"}},{"text":"20: ","color":"green","bold":false},{"text":"[Slime]","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -20"}}]
+execute if score @s particles matches 1.. run tellraw @s [{"text":"21: ","color":"green","bold":false},{"text":"[Rainbow Puffs]  ","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -21"}},{"text":"22: ","color":"green","bold":false},{"text":"[Rainbow Swirls]","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -22"}}]
+execute if score @s particles matches 1.. run tellraw @s [{"text":"23: ","color":"green","bold":false},{"text":"[Snow Cloud]  ","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -23"}},{"text":"24: ","color":"green","bold":false},{"text":"[Smoke]","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -24"}}]
+execute if score @s particles matches 1.. run tellraw @s [{"text":"25: ","color":"green","bold":false},{"text":"[Conduit Eyes]  ","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -25"}},{"text":"26: ","color":"green","bold":false},{"text":"[Nectar]","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -26"}}]
+execute if score @s particles matches 1.. run tellraw @s [{"text":"27: ","color":"green","bold":false},{"text":"[Warped Spores]  ","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -27"}},{"text":"28: ","color":"green","bold":false},{"text":"[Squid Ink]","color":"aqua","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -28"}}]
+execute if score @s particles matches 1.. run tellraw @s [{"text":"29: ","color":"green","bold":false},{"text":"[Tears]  ","color":"gold","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -29"}},{"text":"30: ","color":"green","bold":false},{"text":"[Halo]","color":"gold","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -30"}}]
+execute if score @s particles matches 1.. run tellraw @s [{"text":"50: ","color":"green","bold":false},{"text":"[Olex]  ","color":"dark_red","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -50"}},{"text":"51: ","color":"green","bold":false},{"text":"[Green Notes]","color":"dark_green","bold":false,"clickEvent":{"action":"run_command","value":"/trigger particles set -51"}}]
```

Comment:
Any reason why it skips directly to 50 here? If we want to insert particles in the future we can still change the indices

Reply:
It skipped to 50 because those were originally exclusive ones (the kian, sr mod and green notes ones), so I wanted to separate them. I didn't change the numbers when I made them public. I will do that